vignettes/deploying-dccvalidator.Rmd

```diff
@@ -37,6 +37,14 @@ stand up a new version that is customized for a different community):
 1. If needed, make any changes to the application's behavior by editing the
    files, or check out a branch that contains your changes
 1. To ensure packages are up-to-date, open an R session and run `renv::restore()`
+1. The Sage Shiny Server administrator must configure the utilization scheduler to ensure each connection has its own R process. 
+```
+location /users {
+  location /yourusername/ShinyApps {
```

Comment:
```suggestion
  location /yourusername/yourappname {
```

This seems to be the invocation that actually works, judging by what I currently see in the shiny server config file